<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentsView</class>
 <widget class="QWidget" name="AppointmentsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>就诊记录管理</string>
  </property>
  <widget class="QPushButton" name="btnsave">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>331</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>病人姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboboxPatient"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>医生姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboboxDoctor"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>开具药品：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboboxMedicines"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>就诊日期：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateeditVisit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>诊断结果</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="texteditDiagnosis">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
